src/plugins/walletActions/cpk.ts

```diff
@@ -29,6 +29,9 @@ export const addCPKToBatch = async (syncWallet: Wallet, fee: GweiBalance, feeTok
   let pubKeyTx: CPKLocal | undefined;
   try {
     pubKeyTx = getCPKTx(store.account.address!);
```

Comment:
Strict type instead of typeof condition

Reply:
?

Reply:
Ok, since typeof will cover possible changes of the external lib — here it can be kept